### PR TITLE
adding terms conditions link to NeedHelp.js & OfferHelp.js

### PR DESCRIPTION
**Pull request by rixiobarrios**
### What does this pull request aim to achieve? 💯
Adddind the link for terms and conditions above the email input box for both getHelp and needHelp options/steps
_Please be concise and link any related issue(s):_
#753 

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [x] This branch is rebased/merged with the **latest master**.
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

File: client/src/pages/NeedHelp.js [hunk 1]
```diff
@@ -108,6 +109,10 @@ const Step3 = (props) => {
         Question {props.currentStep}/{props.totalSteps}
       </WizardProgress>
       <StepTitle>What is your email address?</StepTitle>
+      <StyledDiv>
+        We respect your privacy. Please read our{" "}
```
Comment (BMariscal) [inline client/src/pages/NeedHelp.js:113]: I know this was added by the linter, but the `{" "}` shouldn't be here.

File: client/src/pages/OfferHelp.js [hunk 2]
```diff
@@ -133,6 +133,10 @@ const Step3 = (props) => {
         Question {props.currentStep}/{props.totalSteps}
       </WizardProgress>
       <StepTitle>What is your email address?</StepTitle>
+      <StyledDiv>
+        We respect your privacy. Please read our{" "}
```
Comment (BMariscal) [inline client/src/pages/OfferHelp.js:137]: Also here.

File: client/src/pages/NeedHelp.js (hunk 1 above)
Comment (idjevm) [reply client/src/pages/NeedHelp.js:113]: Ohhh actually this was intentional, I reviewed this before Rixio committed. If you remove {" "}, then there won't be space between the our and Link...

File: client/src/pages/OfferHelp.js (hunk 2 above)
Comment (idjevm) [reply client/src/pages/OfferHelp.js:137]: Same comment as above ! :) This should be added if we have a space needed before link.

**Pull request merged**